workshops/kahoot/README.md

```diff
@@ -0,0 +1,106 @@
+---
+name: 'Hacking Kahoot'
+description: 'A learning activity focused around the Kahoot API'
+author: '@sampoder'
+img: 'https://cloud-gl0z7ytn9.vercel.app/0screenshot_2020-12-10_at_9.49.09_pm.png'
+---
+
+Hey! This is an interactive activity for leaders to run with their clubs. 
+It takes them through the process of using the [`kahoot.js-updated` npm module](https://www.npmjs.com/package/kahoot.js-updated) 
+to control a Kahoot user with Node.js. 
+
+A few quick pointers about the workshop:
+
+- The workshop uses the challenge version of Kahoot as that is the most flexible
+- The workshop is a slideshow, so you'll need a lot of energy to make the activity amazing.
+- Be prepared to adapt and help people as they do the workshop
+
+## Preparing
+
+Now, here's a step by step guide on how to prepare for the activity on Kahoot.
+
+First, you'll need to head to [kahoot.com](https://kahoot.com) and create an account.
+
+Head to [this page](https://create.kahoot.it/v2/discover) and choose a Kahoot that you feel will attract your club members.
+
+Click the `Play` button.
+
+![Play button](https://cloud-lewnc6ao9.vercel.app/3screenshot_2020-12-10_at_10.42.35_pm.png)
+
+Choose the option on the right.
+
+![Option to pick](https://cloud-lewnc6ao9.vercel.app/2screenshot_2020-12-10_at_10.44.16_pm.png)
+
+Configure them as per the following image and then click `Create`.
+
+![Config window](https://cloud-lewnc6ao9.vercel.app/1screenshot_2020-12-10_at_10.44.43_pm.png)
+
+Copy the Game PIN provided and keep it safe.
+
+![Game PIN highlighted](https://cloud-lewnc6ao9.vercel.app/0screenshot_2020-12-10_at_10.45.27_pm.png)
+
+Repeat this two more times.
+
+Once you have your game codes you are ready.
+
+## Executing
+
+On the day, load the slideshow from [here](https://workshop-deck-playground-git-add-kahoot-hacking.hackclub.dev/kahoot/0). 
+
+Run the slideshow through, make sure to give the attendees time to type out their code. 
+
+When you reach the `Let's give it a go!` slide, pass your students the Kahoot code and let them run their programs. 
+Load up Kahoot's reports tab and show the student's scores.
+
+Then comes the challenge section. Begin by doing as the slides instruct. However, if you find the students are struggling load up a [repl.it](https://repl.it) 
+repl with the following code:
+
+```javascript
+const Kahoot = require("kahoot.js-updated"); 
```

Comment:
Can you explain a little about what this code does?

Reply:
sure thing